src/cluster/src/cli/start/sys.rs

```diff
@@ -38,11 +38,11 @@ fn install_sys_impl(
     let installer = SysInstaller::from_config(config)?;
 
     if upgrade {
```

Comment:
Seems like we need another mode?
In one mode, we know exactly whether we want install/upgrade. For example, new installation.
In other case, we don't really care.  If sys chart exists, upgrade is done otherwise new sys chart is installed.